src/AzureEventGridSimulator/Settings/SubscriptionSettings.cs

```diff
@@ -9,5 +9,8 @@ public class SubscriptionSettings
 
         [JsonProperty(PropertyName = "endpoint", Required = Required.Always)]
         public string Endpoint { get; set; }
+
+        [JsonProperty(PropertyName = "eventTypes", Required = Required.AllowNull)]
+        public string[] EventTypes { get; set; }
```

Comment:
Someone else has contributed a fairly complete subscription filtering implementation which has now been merged. It would be good to just make this PR soley about dockerisation.